src/platform_impl/windows/monitor.rs

```diff
@@ -162,6 +163,220 @@ impl MonitorHandle {
     }
 }
 
+use std::ffi::OsStr;
+use std::mem::zeroed;
+use std::os::windows::ffi::OsStrExt;
+use std::ptr::null_mut;
+
+use windows_sys::Win32::Foundation::ERROR_SUCCESS;
+use windows_sys::Win32::Graphics::Gdi::{
+    EnumDisplayDevicesW, DISPLAY_DEVICEW, DISPLAY_DEVICE_ACTIVE, DISPLAY_DEVICE_ATTACHED, DMDO_270,
+    DMDO_90,
+};
+use windows_sys::Win32::System::Registry::{
+    RegCloseKey, RegEnumKeyExW, RegOpenKeyExW, RegQueryValueExW, HKEY, HKEY_LOCAL_MACHINE, KEY_READ,
+};
```

Comment:
Nit: Either put these imports at the top, or if you feel it's complex enough, move the EDID logic to a separate file.